master/internal/configpolicy/task_config_policy.go

```diff
@@ -0,0 +1,51 @@
+package configpolicies
```

Comment:
Let's use the singular here. (I know we've gone back and forth a few times, sorry!)